mis_builder/report/mis_report_instance_qweb.xml

```diff
@@ -1,34 +1,28 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <odoo>
 
-    <record id="qweb_pdf_export" model="ir.actions.report.xml">
+    <record id="qweb_pdf_export" model="ir.actions.report">
       <field name="name">MIS report instance QWEB PDF report</field>
       <field name="model">mis.report.instance</field>
-      <field name="type">ir.actions.report.xml</field>
+      <field name="type">ir.actions.report</field>
       <field name="report_name">mis_builder.report_mis_report_instance</field>
       <field name="report_type">qweb-pdf</field>
-      <field name="auto" eval="False"/>
     </record>
-
-    <template id="assets_report" inherit_id="report.assets_common">
-        <xpath expr="." position="inside">
-          <link href="/mis_builder/static/src/css/report.css" rel="stylesheet"/>
-        </xpath>
-    </template>
-
+  
     <!--
         TODO we use divs with css table layout, but this has drawbacks:
              (bad layout of first column, no colspan for first header row),
              consider getting back to a plain HTML table.
     -->
 
     <template id="report_mis_report_instance">
-      <t t-call="report.html_container">
+      <t t-call="web.html_container">
         <t t-foreach="docs" t-as="o">
-          <t t-call="report.internal_layout">
+          <t t-call="web.internal_layout">
             <t t-set="matrix" t-value="o._compute_matrix()"/>
             <t t-set="style_obj" t-value="o.env['mis.report.style']"/>
             <div class="page">
+              <link href="/mis_builder/static/src/css/report.css" rel="stylesheet"/>
```

Comment:
any good reason to keep the css here instead of assets?

Reply:
good question. @jbeficent do you know why this change was made?

Reply:
@simahawk is it better now?

Reply:
@sbidoul should I see any difference? :thinking:

Reply:
@simahawk yes (https://github.com/OCA/mis-builder/pull/67/commits/0237f5944246ac67111cfdbb220f87a60028ce63). Apparently github has difficulties showing the diff inside the conversation. But the full diff is correct.

Reply:
ahh now I see it. LGTM :smile: